pleskdistup/common/src/leapp_configs.py

```diff
@@ -33,7 +33,8 @@
 """
 
 
-def _do_replacement(to_change: str, replacement_list: typing.List[typing.Callable[[str], str]]) -> typing.Optional[str]:
+def _do_replacement(to_change: typing.Optional[str], replacement_list: typing.List[typing.Callable[[str], str]]
```

Comment:
Please use semantic indentation, putting a newline immediately after opening parenthesis or bracket, like
```
(
    x,
    y,
    [
        z,
    ],
)
```
Grouping short values is fine too:
```
(
    a, b,
    [ c, d ],
)
```
On the other hand, visual indentation wastes too much precious horizontal space, like
```
def extract_repodata(repofile: str) -> typing.Iterable[typing.Tuple[typing.Optional[str], typing.Optional[str],
                                                                    typing.Optional[str], typing.Optional[str],
                                                                    typing.Optional[str], typing.List[str]]]:
```
would be better as
```
def extract_repodata(
    repofile: str,
) -> typing.Iterable[
    typing.Tuple[
        typing.Optional[str],
        typing.Optional[str],
        typing.Optional[str],
        typing.Optional[str],
        typing.Optional[str],
        typing.List[str],
    ]
]:
```